packages/contracts-bedrock/scripts/slither.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ ! -d forge-artifacts/build-info ]; then
+    npx hardhat compile
```

Comment:
I tested and this does seem to work, but for my own info, how does `npx hardhat compile` generate files in `forge-artifacts/build-info`?

Reply:
`foundry` outputs the build info directory in the artifacts directory || `build-info` and isn't configurable currently, i'd like to add in a way to configure the outdir and then we could just configure it to go in the same directory that hh outputs them